test/filters/all.test.js

```diff
@@ -79,11 +141,11 @@ describe('Filters', () => {
 
   describe('.renderSpec', () => {
     it('should work', async () => {
-      const doc = new AsyncAPIDocument({ asyncapi: '2.0.0', info: { title: 'dummy spec for testing', version: '1.5.73' } });
+      const doc = new AsyncAPIDocument({ asyncapi: '2.0.0', info: { title: 'dummy spec for testing', version: '2.1.37' } });
 
       const result = renderSpec(doc);
-      // check if '1.5.73' version is rendered
-      expect(result.includes('1.5.73')).toEqual(true);
+      // check if '2.1.37' version is rendered
+      expect(result.includes('2.1.37')).toEqual(true);
       // check if 'dummy spec for testing' title is rendered
```

Comment:
Why was this changed? 🤔

Reply:
Do you not like the `2.1.37`? 😄 It's only a test data.